README.md

```diff
@@ -1,2 +1,22 @@
 # edge-common-spring
-a shared library/framework for edge APIs for spring way modules
+
+Copyright (C) 2021 The Open Library Foundation
+
+This software is distributed under the terms of the Apache License, Version 2.0. See the file "LICENSE" for more information.
+
+## Introduction
+
+This is a shared library/framework for edge APIs for spring way modules.
```

Comment:
It's worth mentioning that this is a port of edge-common (with link) and that the goal is feature parity with that framework.  Edge modules developed by either should be indistinguishable from a hosting/SysOps perspective.